packages/web-comment/src/routes/appeals/_appealNumber/ip-application-rows.js

```diff
@@ -14,11 +14,6 @@ const applicationRows = (caseData) => {
 	const documents = caseData.Documents || [];
 
 	return [
-		{
-			keyText: 'Planning application number',
-			valueText: caseData.LPAApplicationReference,
-			condition: (caseData) => caseData.LPAApplicationReference
```

Comment:
A suggestion I got on the questionnaire page was not passing in the data for the condition of every row and just using case data from the initial argument.
for each condition I just did `() => caseData.LPAApplicationReference` for example

Reply:
Thanks, made the change